src/plays/dev-jokes/DevJokes.jsx

```diff
@@ -0,0 +1,40 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import "./devJokes.css";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+function DevJokes(props) {
+  const [joke, setjoke] = useState([]);
+  const fetch = () => {
+    axios
+      .get("https://backend-omega-seven.vercel.app/api/getjoke")
+      .then((response) => {
+        setjoke(response.data);
+      });
+  };
+  useEffect(() => {
+    fetch();
+  }, []);
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="maincontanier">
+          <div className="jokecontanier">
+            {joke.map((value) => {
+              return (
+                <div key={value}>
```

Comment:
A few of the jokes appear `center` aligned and a few are `left` aligned. Can you control those to make a uniform experience?

Reply:
yeah, I saw that it can be repaired by resizing the jokecont but  it's making the text uenven

Reply:
yeah, I saw that it can be repaired by resizing the jokecont but it's making the text uneven